website/bitcoin_utils.py

```
@@ -58,3 +60,51 @@ def issue_asset(asset_name, amount, identifier):
+    formatted_amount = f"{Decimal(amount):.8f}"
+
+    url = settings.BCH_PAYMENT_API_URL
+
+    payload = {"to_address": address, "amount": formatted_amount, "currency": "BCH"}
+
+    headers = {"Authorization": f"Bearer {settings.BCH_API_KEY}", "Content-Type": "application/json"}
+
+    try:
+        response = requests.post(url, json=payload, headers=headers, timeout=10)
+    except requests.exceptions.RequestException as e:
+        logger.error(f"BCH payment request failed: {str(e)}")
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)